Platform/OpenCanopy/Views/BootPicker.c

```diff
@@ -413,7 +418,11 @@ InternalBootPickerEntryDraw (
   ASSERT (Context != NULL);
 
   Entry       = BASE_CR (This, GUI_VOLUME_ENTRY, Hdr.Obj);
-  EntryIcon   = Entry->EntryIcon;
+  /*if (mBootPickerImageIndex < 5) {
+    EntryIcon = &((BOOT_PICKER_GUI_CONTEXT *) DrawContext->GuiContext)->Poof[mBootPickerImageIndex];
+  } else */{
+    EntryIcon   = Entry->EntryIcon;
+  }
```

Comment:
Does the commented out code above make sense?

Reply:
It's for testing Poof animation.